### ISSUE TITLE
 [a11y]: Combobox [default] - screen reader bug with options

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#15260]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.40.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.39.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Description
SR is reading Option 1  of 3 OR 2 of 3 when we navigate using arrow keys

SR is reading Option 1  of 3 OR 2 of 3 when we navigate using arrow keys. We understood there is 3rd option which is disabled where in SR users are not able to reach using Jaws in arrow keys.

### Recommended Fix
When the state of an element i.e. disabled, selected, expanded, currently active etc. is not conveyed for the screen reader user, it can be misleading for the user. . this impacts the user interaction.

### WCAG 2.1 Violation
4.1.2 Name, Role, Value

**Comment by 2nikhiltom** [comment]
Re-tested , issue is confirmed on v11.84.0 as well
